src/commands/deploy.js

```diff
@@ -16,8 +14,8 @@ const isObject = require('lodash.isobject')
 const SitesCreateCommand = require('./sites/create')
 const LinkCommand = require('./link')
 const { NETLIFYDEV, NETLIFYDEVLOG, NETLIFYDEVERR } = require('../utils/logo')
-
-const statAsync = promisify(fs.stat)
+const { statAsync } = require('../lib/fs')
```

Comment:
We have a few places in the code that we used `promisify` with `fs`.
Moved those to a shared lib